uint16 gpioId
---
uint8 level
bool hasSucceeded